ingestion/src/metadata/ingestion/source/dashboard/tableau/metadata.py

```diff
@@ -375,15 +377,21 @@ def yield_dashboard_chart(
                 if filter_by_chart(self.source_config.chartFilterPattern, chart.name):
                     self.status.filter(chart.name, "Chart Pattern not allowed")
                     continue
-                workbook_name = dashboard_details.name.replace(" ", "")
                 site_url = (
                     f"site/{self.service_connection.siteUrl}/"
                     if self.service_connection.siteUrl
                     else ""
                 )
-                chart_url = (
-                    f"#/{site_url}" f"views/{workbook_name}/" f"{chart.view_url_name}"
-                )
+                chart_url = None
+                if (
+                    "/sheets/" in chart.content_url
+                    and len(chart.content_url.split("/")) > 1
```

Comment:
Instead of calling the `split` method repeatedly it might be better to store in a variable and re-use it

Reply:
Done